### PR TITLE
Update json ld #5692fy

### PR DESCRIPTION
**Pull request by matthewtgilbride**
# Description

Update JSON-LD metadata for datasets.

## Type of change

Delete those that don't apply.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Open a dataset and inspect the `head` tag of the page.  You should see the following:

`<meta name="DC...." content="..." />` for each field, and a script tag `<script vmid="ldjson-schema" ...`

Open the home page, you should see similar meta tags.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: pages/index.vue [hunk 1]
```diff
@@ -72,6 +72,47 @@ export default {
       heroButtonLink: '',
       heroButtonLabel: ''
     }
+  },
+
+  head() {
+    return {
+      title: 'SPARC Portal',
+      titleTemplate: '%s - SPARC Portal',
```
Comment (matthewtgilbride) [inline pages/index.vue]: here

File: pages/index.vue [hunk 2]
```diff
@@ -72,6 +72,47 @@ export default {
...
+          name: 'twitter:card',
+          content: 'summary'
+        },
+        {
+          name: 'twitter:site',
+          content: '@sparc'
+        },
+        {
+          name: 'twitter:description',
+          content: 'Stimulating Peripheral Activity to Relieve Conditions (SPARC)'
```
Comment (matthewtgilbride) [inline pages/index.vue]: here

File: pages/index.vue [hunk 3]
```diff
@@ -72,6 +72,47 @@ export default {
...
+  },
+
+  head() {
+    return {
+      title: 'SPARC Portal',
+      titleTemplate: '%s - SPARC Portal',
+      meta: [
+        {
+          name: 'description',
+          content: 'Stimulating Peripheral Activity to Relieve Conditions (SPARC)'
```
[the remark this answers is not in the document]
Comment (jwagenaar) [reply pages/index.vue]: I think this is the summary of the dataset. I would look at an example on Discover:
https://discover.blackfynn.com/datasets/29

<meta data-vue-meta="1" name="DC.title" content="Molecular Phenotype Distribution of Single Rat Intracardiac Neurons">
<meta data-vue-meta="1" name="DC.description" content="Images collected from serial cryostat sectioning of a cryopreserved heart was used to reconstruct the 3D context. Transcriptional profiles taken from isolated single neurons and mapped back into the previously generated 3D context.">

File: pages/index.vue [hunk 4]
```diff
@@ -72,6 +72,47 @@ export default {
...
+        },
+        {
+          name: 'og:type',
+          content: 'website'
+        },
+        {
+          name: 'og:title',
+          content: 'SPARC Portal'
```
[the remark this answers is not in the document]
Comment (jwagenaar) [reply pages/index.vue]: see above: Needs to be title of the dataset

File: pages/index.vue (hunk 2 above)
Comment (jwagenaar) [reply pages/index.vue]: summary of dataset:
example:
<meta data-vue-meta="1" name="twitter:description" content="Images collected from serial cryostat sectioning of a cryopreserved heart was used to reconstruct the 3D context. Transcriptional profiles taken from isolated single neurons and mapped back into the previously generated 3D context.">

File: pages/datasets/_datasetId.vue [hunk 5]
```diff
@@ -637,9 +637,93 @@ export default {
...
+          property: 'og:image',
+          content: this.getDatasetImage
+        },
+        {
+          property: 'og:image:alt',
+          content: `${this.datasetTitle} Banner Image`
+        },
+        {
+          property: 'og:site_name',
+          content: 'SPARC Portal'
```
[the remark this answers is not in the document]
Comment (jwagenaar) [reply pages/datasets/_datasetId.vue]: I think that is okay

File: pages/index.vue (hunk 4 above)
[the remark this answers is not in the document]
Comment (jwagenaar) [reply pages/index.vue]: Needs to be title of dataset

File: pages/index.vue (hunk 3 above)
[the remark this answers is not in the document]
Comment (matthewtgilbride) [reply pages/index.vue]: sorry I should have made more clear:

`pages/index.vue` is the home page - so everything in this file is for the entire portal

**Pull request merged**